mat/triband.go

```diff
@@ -351,3 +351,17 @@ func (t *TriBandDense) DiagView() Diagonal {
 		},
 	}
 }
+
+// Trace returns the trace.
+func (t *TriBandDense) Trace() float64 {
+	rb := t.RawTriBand()
+	tr := 0.0
```

Comment:
And here for `tr` and `offsetIndex`; `var off int`.

Reply:
fair enough my bad.  all resolved now (i hope!).